src/components/common/Actions.vue

```diff
@@ -1,45 +1,46 @@
 <template>
   <v-container id="action-buttons-container" class="list-item">
-
-    <div class="buttons-left">
-      <!-- disable Save button for now -->
-      <v-btn id="save-btn" large
-        :disabled="isSaveButtonDisabled"
-        :loading="isSaving"
-        @click="onClickSave()"
-      >
-        <span>Save</span>
-      </v-btn>
-
-      <!-- disable Save and Resume Later button for now -->
-      <v-btn id="save-resume-btn" large
-        :disabled="isSaveResumeButtonDisabled"
-        :loading="isSavingResuming"
-        @click="onClickSaveResume()"
-      >
-        <span>Save and Resume Later</span>
-      </v-btn>
-    </div>
-
-    <div class="buttons-right">
-      <v-fade-transition hide-on-leave>
-        <v-btn id="file-pay-btn" large color="primary"
-          :disabled="isFilePayButtonDisabled"
-          :loading="isFilingPaying"
-          @click="onClickFilePay()"
+    <!-- don't show buttons until Entity Type is identified -->
+    <template v-if="isEntityType">
```

Comment:
This looks like a big change but it's mostly whitespace due to adding this template tag.